src/Module/Configuration/Carriers.php

```diff
@@ -468,11 +479,18 @@ private function getList()
         $helper->colorOnBackground = true;
         $helper->no_link = true;
 
-        $list = Db::getInstance()->executeS('SELECT a.*
-            FROM `' . _DB_PREFIX_ . 'carrier` a
-            WHERE (a.external_module_name = \'' . $this->module->name . '\') AND a.`deleted` = 0
-            ORDER BY a.`position` ASC
-            LIMIT 0, 50');
+        $table = Table::withPrefix('carrier');
+        $list = Db::getInstance()
+            ->executeS(
+                <<<SQL
+SELECT *
```

Comment:
Is it known which attributes are in this table? You should write them out if so, this makes refactoring in the future way more easy as it shows in you IDE when you global search.

Reply:
Zal het noteren voor later